flatgeobuf/flatgeobuf.ipynb

```diff
@@ -15,15 +15,13 @@
    "source": [
```

Comment:
<p>just one typo: earlie to earlier or prior</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks, edited!